package.json

```diff
@@ -53,7 +53,10 @@
     "husky": "^0.14.3",
     "jest": "^22.0.4",
     "mock-fs": "^4.4.2",
-    "prettier": "1.9.2",
+    "prettier": "1.13.0",
     "semantic-release": "^11.0.2"
+  },
+  "jest": {
```

Comment:
This should be fixed in newer versions of Jest, right?

Reply:
Are you asking if Jest 23.6.0 has this issue? Because it does.